sparse/rest/server.go

```diff
@@ -2,15 +2,60 @@ package rest
 
 import (
 	"context"
+	"net"
 	"net/http"
 	"os"
+	"sync"
+	"time"
 
 	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
 
 	"github.com/longhorn/sparse-tools/sparse"
 )
 
+var (
+	defaultIdleTimeout = 90 * time.Second
+)
+
+type IdleTimeer struct {
+	mu          sync.Mutex
+	active      map[net.Conn]bool
```

Comment:
nit:
make the naming more readable and understandable, like.
```suggestion
	activeConns      map[net.Conn]bool
```

Reply:
fixed, thanks